### PR TITLE
meta(feedback): Add name to feedback blueprint

### PR DESCRIPTION
**Pull request by billyvg**
* contact_email is optional
* add name as part of feedback obj

### CODE REVIEW

File: src/sentry/feedback/blueprints/api.md [hunk 1]
```diff
@@ -262,8 +262,9 @@ See https://develop.sentry.dev/sdk/event-payloads/types/ for more information
 | environment                | optional[string] | -                                     |
 | event_id                   | optional[string] | Omitted IDs are internally generated. |
 | feedback                   | object           | -                                     |
-| feedback.contact_email     | string           | -                                     |
+| feedback.contact_email     | optional[string] | -                                     |
 | feedback.message           | string           | -                                     |
+| feedback.name              | optional[string] | -                                     |
```
Comment (billyvg) [inline src/sentry/feedback/blueprints/api.md:267]: cc @cmanallen @michellewzhang

Comment (cmanallen) [reply src/sentry/feedback/blueprints/api.md:267]: 👍

Comment (michellewzhang) [reply src/sentry/feedback/blueprints/api.md:267]: i'll add it to the endpoint!

Comment (michellewzhang) [reply src/sentry/feedback/blueprints/api.md:267]: https://github.com/getsentry/sentry/pull/57523

**Review by cmanallen — APPROVED** [review]

**Pull request merged**